### PR TITLE
Add way to inform users a recommended or forced update is available

### LINKED ISSUE #32: Add way to inform users a recommended or forced update is available
Do we want this? The idea is to add information about the latest versions to the discovery json so we can inform the user that they need to update. Alternatively we can also just rely on the update process of the App Stores, though then you can never 'kill' any versions of the app that may have issues.

### PR DESCRIPTION
**Pull request by johankool**
Fixes #32

### CODE REVIEW

File: geteduroam/GeteduroamPackage/Sources/Main/MainView.swift [hunk 1]
```diff
@@ -201,6 +223,23 @@ struct MainContentView: View {
                     break
                 }
             }
+            .onTapGesture(count: 10) {
+                #if canImport(UIKit)
+                let content = UIPasteboard.general.string
+                #elseif canImport(AppKit)
+                let content = NSPasteboard.general.string(forType: .string)
+                #endif
+                if content == "geheim" {
```
Comment (alexwegeniq) [inline geteduroam/GeteduroamPackage/Sources/Main/MainView.swift:232]: "geheim" ?. Can this be a constant instead ?

File: Config/appconfig.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  # Application state for app life management
+  appState: active # active, updateRecommended, updateRequired, updateOSRecommended, updateOSRequired, obsolete, disabled
+
+  # Enable when the API is undergoing maintenance
+  maintenance: false
+
+# Keep track of keys that are no longer in use.
+deprecatedKeys: []
+
+# Override settings
```
Comment (alexwegeniq) [inline Config/appconfig.yaml:12]: Missing description why this is commented out.

File: geteduroam/GeteduroamPackage/Sources/Main/AppStateView.swift [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        case .updateRequired, .updateOSRequired, .disabled:
+            true
+        }
+    }
+    
+    var systemName: String {
+        #if canImport(UIKit)
+        UIDevice.current.systemName
+        #else
+        "macOS"
```
Comment (alexwegeniq) [inline geteduroam/GeteduroamPackage/Sources/Main/AppStateView.swift:27]: This might fail if Apple released a product which is not a Mac or running iOS.

**Review by risinggalaxy — APPROVED** [review]

**Review by alexwegeniq — APPROVED** [review]
Minor stuff

**Pull request merged**